docs/specification.rst

```diff
@@ -348,7 +348,7 @@ Slice expressions adhere to the following rules:
 * If no stop position is given, it is assumed to be the length of the array if
   the given step is greater than 0 or 0 if the given step is less than 0.
 * If the given step is omitted, it it assumed to be 1.
-* If the given step is 0, an error MUST be raised.
+* If the given step is 0, an ``invalid-value`` error MUST be raised.
```

Comment:
Let's hold off on `invalid-value` for now.  I agree that we should add a specific type of error, but given the spec never required raising a specific type, we can only make a recommendation of the specific error type for this case.  We also don't have an `invalid-value` type defined for the spec so we should figure out if that's an error type we want to add first (I do see the PR about adding a section on error types).